specification/compute/resource-manager/Microsoft.Compute/DiskRP/stable/2023-01-02/diskRPCommon.json

```diff
@@ -542,6 +542,29 @@
         "getSecureVMGuestStateSAS": {
           "type": "boolean",
           "description": "Set this flag to true to get additional SAS for VM guest state"
+        },
+        "fileFormat": {
+          "type": "string",
+          "description": "Used to specify the file format when making request for SAS on a VHDX file format snapshot",
+          "enum": [
+            "None",
+            "VHD",
+            "VHDX"
+          ],
+          "x-ms-enum": {
+            "name": "FileFormat",
```

Comment:
please have a test for this, look at examples folder

Reply:
Added